### PR TITLE
Update content in homepage salary component

### PR DESCRIPTION
**Pull request by MaxineCl**
### Trello card
https://trello.com/c/44SRZhfh/5480-tweak-content-in-salary-component-on-homepage-to-align-with-campaign-messaging-language

### Context
We need to make updates to the website to align messaging with the new creative campaign launching on 15 January 2024

### Changes proposed in this pull request
Tweak the wording in the salary component to align with the wording in the messaging matrix.

### CODE REVIEW

**Review by caitlin-reidy — COMMENTED** [review]
Looks good to me!

File: app/views/content/home/_salaries-banner.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div id="salary-banner" class="banner">
   <%= render CallsToAction::SimpleComponent.new(icon: "icon-money") do %>
-    <h2 class="call-to-action__heading">Earn at least &pound;30k a year</h2>
-    <p>All qualified teachers will have a <strong>starting salary of at least £30,000</strong> (or higher in London).</p>
+    <h2 class="call-to-action__heading">Earn a minimum of &pound;30k a year</h2>
+    <p>All qualified teachers get a <strong>minimum starting salary of £30k</strong> (or higher in London). So it pays to do what you love. And with lots of opportunities to progress, you could <strong>earn over £41k within 5 years</strong>.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/home/_salaries-banner.html.erb:4]:
```suggestion
    <p>All qualified teachers get a <strong>minimum starting salary of £30k</strong> (or higher in London), so it pays to do what you love. And with lots of opportunities to progress, you could <strong>earn over £41k within 5 years</strong>.</p>
```
Would this be better as one sentence? But happy for you to disagree

Comment (gemmadallmandfe) [reply app/views/content/home/_salaries-banner.html.erb:4]: Also - I wonder if the `And with lots of opportunities....` sentence might be better on a new line - might make the 30k and 41k stand out more?

Again, happy for you to disagree!

**Review by gemmadallmandfe — COMMENTED** [review]
Really like this @MaxineCl !

Have left a couple of suggestions but happy to take your steer

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**